src/main/java/CarGame2/domain/model/Car.java

```diff
@@ -0,0 +1,38 @@
+package CarGame2.domain.model;
+
+public class Car {
+
+    private static final int MIN_SPEED = 4;
+    private static final int MAX_NAME_LENGTH = 5;
+    private final String name;
+
+    private int position = 0;
```

Comment:
```suggestion
    private static final int MIN_SPEED = 4;
    private static final int MAX_NAME_LENGTH = 5;
    
    private final String name;
    private int position = 0;
```

필드와 상수는 개행으로 구분되면 더욱 좋을 것 같아요!